### DISCUSSION TITLE
Solidity code in Github discussion

### DISCUSSION THREAD

**Discussion by malaidev** [smartcontractkit/full-blockchain-solidity-course-py#1766] [General]
A few days ago, I developed a new solidity based smart contract and deployed.
I hope to join in the github discussion to talk about experiences to overcome challenges.
Thank you

**Comment by cryptoCoder0302** [comment]
Perfect!

**Comment by bolo821** [comment]
Excellent!😃

**Comment by whitetigerhunter** [comment]
Good code!

**Comment by atahabilder1** [comment]
I have recently started to learn this course. It is a perfect resource.
